Makefile

```diff
@@ -17,7 +17,7 @@
 VERSION ?= `cat $(PWD)/VERSION`
 # Default bundle image tag
 BUNDLE_IMG ?= quay.io/devfile/registry-operator-bundle:v$(VERSION)
-CERT_MANAGER_VERSION ?= v1.11.0
+CERT_MANAGER_VERSION ?= v1.13.1
```

Comment:
Where are we finding `v1.13.1` as the latest? I am looking here https://cert-manager.io/docs/releases/ and see `1.15` as the latest with `1.12` as LTS.

Do we want the latest or something with LTS? @michael-valdron @thepetk

Reply:
@Jdubrick I think is a fair point. I'd suggest the latest version that doesn't introduce breaking changes affecting our operateor. Looking at the same link you've shared I see that the suggested version `1.13.1` is no longer supported. As a result, if possible I'd suggest going for >1.14 version as they are supported.